### PR TITLE
Add dependencies between more of the devstack services so that they s…

### PR DESCRIPTION
**Pull request by cpennington**
…tart in the right order

### CODE REVIEW

**Review by crice100 — COMMENTED** [review]
Are these changes already included in https://github.com/edx/devstack/pull/419?

**Comment by cpennington** [comment]
Yes.

File: docker-compose.yml [hunk 1]
```diff
@@ -150,6 +152,10 @@ services:
       - mysql
       - memcached
       - mongo
+      - discovery
+      - forum
+      - firefox
```
Comment (robrap) [inline docker-compose.yml:157]: Do we need the browsers?

**Review by robrap — APPROVED** [review]
One question.

**Review by crice100 — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (cpennington) [reply docker-compose.yml:157]: Yeah, if you want to run bok_choy tests. I didn't want to add another barrier to being able to run those tests out of the box.

**Pull request merged**